src/traits.rs

```diff
@@ -62,7 +62,7 @@ impl<'a, 'tcx> EvalContext<'a, 'tcx> {
 
         for (i, method) in ::rustc::traits::get_vtable_methods(self.tcx, trait_ref).enumerate() {
             if let Some((def_id, substs)) = method {
-                let instance = ::eval_context::resolve(self.tcx, def_id, substs);
+                let instance = resolve(self.tcx, def_id, substs);
```

Comment:
I plan on removing them from trans once miri is merged into rustc, because the trans versions have nothing llvm related in them.

Do you mean that I should import the module in general or just for things that are copied from trans?